File: jApp/src/main/java/j/s/yarlykov/data/network/model/WeatherModel.java
package j.s.yarlykov.data.network.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class WeatherModel {
    @SerializedName("id") public int id;
    @SerializedName("main") public String main;
    @SerializedName("description") public String description;
    @SerializedName("icon")
    @Expose
    public String icon;
}

File: kApp/src/main/java/k/s/yarlykov/data/network/model/WeatherModel.kt
package k.s.yarlykov.data.network.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class WeatherModel {
    @SerializedName("id")
    var id: Int = 0
    @SerializedName("main")
    var main: String? = null
    @SerializedName("description")
    var description: String? = null
    @SerializedName("icon")
    @Expose
    var icon: String? = null
}